templates/blog/post_list.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% for post in post_list %}
+    <div class="post">
+      <p><b>Title</b> : <a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></p>
+      <p>{{ post.owner }} at {{ post.created_time }}</p>
+      <p><b>Content</b> : {{ post.content }}</p>
```

Comment:
Consider limiting the amount of content displayed in the list view. Showing the entire content of a post might make the page very long. You could use Django's `truncatewords` or `truncatechars` template filters to limit the length of the content.